politeiad/cache/cockroachdb/cockroachdb.go

```diff
@@ -827,7 +827,7 @@ func New(user, host, net, rootCert, cert, key string) (*cockroachdb, error) {
 	// names manually.
 	c.recordsdb.SingularTable(true)
 
-	log.Infof("Cache host: %v", h)
+	log.Infof("Cache host  : %v", h)
```

Comment:
It's so that it lines up with the userdb host output.

```
2019-05-07 11:38:53.083 [INF] CODB: UserDB host : postgresql://politeiawww@localhost:26257/users_testnet3
2019-05-07 11:38:53.277 [INF] CODB: Cache host  : postgresql://politeiawww@localhost:26257/records_testnet3
```

Get rid of it?